android/src/main/java/com/izo/rnadmob/RNPublisherBannerViewManager.java

```diff
@@ -221,12 +242,13 @@ public void onAppEvent(String name, String info) {
 
 public class RNPublisherBannerViewManager extends ViewGroupManager<ReactPublisherAdView> {
 
-    public static final String REACT_CLASS = "RNGADBannerView";
+    public static final String REACT_CLASS = "RNGAMBannerView";
```

Comment:
Why do we need to rename it?

Reply:
Due to RNPublisherBanner.js containing that component name. For some weird reason the name wasn't changed here.  Not changing it breaks the container on Android.